fullstack/README.md

```diff
@@ -26,4 +29,6 @@ current code.
   - Tests
   - Business logic
   - Design choices
-- It's not perfect, we'd love to hear how you'd improve it
+- We had fun creating it. It's not perfect, we'd love to hear how you'd improve it.
```

Comment:
```suggestion
  - We had fun creating it. It's not perfect, we'd love to hear how you'd improve it.
```
Can we please fix this alignment ?

Reply:
That line is aligned with line 27 and was not in the same group with the ones directly above.